disnake/guild.py

```diff
@@ -286,6 +304,8 @@ class Guild(Hashable):
         "_stage_instances",
         "_threads",
         "_application_commands",
+        "approximate_member_count",
+        "approximate_presence_count",
```

Comment:
Maybe we keep this alphabetical?

Reply:
The list doesn't seem to be alphabetical, but I suggest adding those two lines below the "nsfw" one.

Reply:
it's underneath the other ones, since it's new.